packages/orchestration/src/typeGuards.js

```diff
@@ -3,11 +3,21 @@ import { VowShape } from '@agoric/vow';
 import { M } from '@endo/patterns';
 
 /**
- * @import {TypedPattern} from '@agoric/internal';
+ * @import {TypedPattern as TPInternal} from '@agoric/internal';
```

Comment:
why rename this? If `TypedPattern` is to be internal (which I support) it should simply have the `@internal` tag. For that matter, I suppose everything in `@agoric/internal` should.

Reply:
The problem was: The docs showed `DelegationShape: TypedPattern<Delegation>` but `TypedPattern` wasn't linked, because it was from the internal package.

So this is an alias in the orchestration so that it gets linked. For example, see:
https://dc-orch-ref-docs.agoric-sdk.pages.dev/variables/_agoric_orchestration.DelegationShape

Moving `TypedPattern` to `@endo/patterns` might be a better way to fix this, but I'm not sure how to test that quickly.

Reply:
can we leave it unlinked? if `Delegation` is linked, that seems enough to me

Reply:
> can we leave it (TypedPattern) unlinked?

I'm making a separate issue about it; we can decide to close it without action:
 - #9994